docs/reference/search/point-in-time-api.asciidoc

```diff
@@ -57,7 +57,7 @@ POST /_search  <1>
 // TEST[catch:unavailable]
 
 <1> A search request with the `pit` parameter must not specify `index`, `routing`,
-and {ref}/search-request-body.html#request-body-search-preference[`preference`]
+and {ref}/search-search.html[`preference`]
```

Comment:
Unless the content is being used in another docbook, it's better to use internal cross-references.
```suggestion
or <<search-preference,`preference`>>
```

Reply:
Thank you for catching that